lib/commons/dom/is-visible.js

```diff
@@ -226,6 +209,15 @@ function isVisible(el, screenReader, recursed) {
     return false;
   }
 
+  if (nodeName === 'summary') {
+    return true;
+  }
+
+  // Handle lack of support in IE
+  if (nodeName === 'details' && 'open' in el && !el.hasAttribute('open')) {
+    return false;
```

Comment:
I think maybe here we need to look at the nodeName of the parent, rather than the current node. The details element isn't hidden when it's closed. There are all sorts of ways in which that can make a difference, for example if there's an invalid ARIA attribute on a closed details element, we'll still want to report that I think.